bridge/setu/processor/checkpoint.go

```diff
@@ -3,14 +3,14 @@ package processor
 import (
 	"context"
 	"encoding/hex"
-	"encoding/json"
 	"errors"
 	"math"
 	"math/big"
 	"strconv"
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	jsoniter "github.com/json-iterator/go"
```

Comment:
Do we store json anywhere in Heimdall into blockchain?

Reply:
Not that I am aware of

Reply:
Ok. We'll see it in tests and deploy.